dirty_cat/minhash_encoder.py

```diff
@@ -47,6 +47,9 @@ class MinHashEncoder(BaseEstimator, TransformerMixin):
     handle_missing : 'error' or 'zero_impute' (default)
         Whether to raise an error or encode missing values (NaN) with
         vectors filled with zeros.
+    n_jobs : int, default=-1
```

Comment:
The default n_jobs should be `None` as inspired from [here (Jo Blib)](https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html)